src/components/Router/index.js

```diff
@@ -137,25 +145,34 @@ class MainRouter extends Component {
     super(props)
     this.state = {
       crossDeviceInitialStep: null,
+      i18n: this.initializeI18n()
     }
   }
 
   mobileConfig = () => {
     const {documentType, options} = this.props
-    const {steps, token} = options
+    const {steps, token, language} = options
     const woopraCookie = getWoopraCookie()
-    return {steps, token, documentType, step: this.state.crossDeviceInitialStep, woopraCookie}
+    return {steps, token, language, documentType, step: this.state.crossDeviceInitialStep, woopraCookie}
   }
 
   onFlowChange = (newFlow, newStep, previousFlow, previousStep) => {
     if (newFlow === "crossDeviceSteps") this.setState({crossDeviceInitialStep: previousStep})
   }
 
+  initializeI18n = () => {
+    const {actions, options} = this.props
+    const i18n = setI18n(options.language)
+    actions.setTranslations(i18n.phrases)
```

Comment:
What's the rationale between putting the phrases in the store vs. just the locale e.g. `'en'`?

Reply:
We need to send the phrases to the cross device. This is in preparation of my next ticket in which we will allow for custom translations.

Reply:
but can't get the phrases from the `polyglot` instance when you need to send them to the cross device? Why do they need to go in the store?

Reply:
Yeah, I will separate it from this ticket actually. It doesn't make sense to have it here.

Reply:
Done.